app/controllers/reviews_controller.rb

```diff
@@ -12,6 +12,8 @@ def create
     @rating_collection = [1, 2, 3, 4, 5]
 
     if @review.save
+      # binding.pry
```

Comment:
delete these comments before pushing (better yet just delete rather than comment them in general) ;)